ert_logging/__init__.py

```diff
@@ -1,3 +1,13 @@
 import pathlib
+import os
+from logging import FileHandler
+from datetime import datetime
 
 LOGGING_CONFIG = pathlib.Path(__file__).parent.resolve() / "logger.conf"
+
+
+class TimestampedFileHandler(FileHandler):
+    def __init__(self, filename, *args, **kwargs):
+        filename, extension = os.path.splitext(filename)
+        filename = f"{filename}-{datetime.now().strftime('%Y-%m-%d-%s')}{extension}"
```

Comment:
add hour and minute as well, for easier identification

Reply:
Agreed, change to: `'%Y-%m-%dT%H-%M'` which should be ISO8601 compatible